packages/playground/src/calculator/resource_pricing.vue

```diff
@@ -230,32 +235,61 @@ const calculator = computed(() => {
     ? gridStore.grid?.calculator
     : new Calculator(new QueryClient(window.env.SUBSTRATE_URL));
 });
-watch([CRU, MRU, SRU, HRU, balance, isCertified, ipv4, currentbalance, hasActiveProfile], async () => {
-  let pkgs: any;
-  if (!valid.value.error) {
-    if (currentbalance.value && hasActiveProfile.value && gridStore.grid) {
-      const accountBalance = await gridStore.grid?.balance.getMyBalance();
-      balance.value = accountBalance?.free;
+watch(
+  [CRU, MRU, SRU, HRU, balance, isCertified, ipv4, currentbalance, hasActiveProfile],
+  async () => {
+    let pkgs: any;
+    if (!valid.value.error) {
```

Comment:
we can avoid those errors
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/54a092b2-51ff-4f1f-9324-a31527629d5f)
```suggestion
    if (!valid.value?.error) {
```